### DISCUSSION TITLE
How to blacklist cache inputs?

### DISCUSSION THREAD

**Discussion by cjayyy** [vercel/turborepo#1243] [Help] [answered]
Hi 👋 

In [caching docs](https://turborepo.org/docs/core-concepts/caching#alter-caching-based-on-environment-variables-and-files) I found an option how to include specific inputs (files) in task caching. **Is it also possible to exclude inputs** instead?

**Comment by anthonyshew** [comment, marked as answer]
You can specify ignore an input glob using `!`. For example, to ignore a README.md file:
```
    "inputs": ["!README.md"]
```

https://turbo.build/repo/docs/reference/configuration#inputs

**Reply by cjayyy** [reply]
Thank you 👍
